### PR TITLE
Document event sequence attribute

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Adds sequence to examples in events API doc
- Adds sequence to list of event contents and description tables in event spec in the events reference doc

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues

### CODE REVIEW

File: content/sensu-go/6.2/observability-pipeline/observe-events/events.md [hunk 1]
```diff
@@ -376,6 +379,13 @@ required     | false
 type         | String
 example      | {{< code shell >}}"event_id": "431a0085-96da-4521-863f-c38b480701e9"{{< /code >}}
 
+sequence     |      |
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/observability-pipeline/observe-events/events.md:384]:
```suggestion
<a name="sequence-attribute"></a>

sequence     |      |
```

File: content/sensu-go/6.2/api/events.md [hunk 2]
```diff
@@ -30,7 +30,8 @@ HTTP/1.1 200 OK
 [
   {
     "timestamp": 1542667666,
-    "id": "caaf2c38-2afb-4f96-89b3-8ca5c3e6f449",
+    "event_id": "caaf2c38-2afb-4f96-89b3-8ca5c3e6f449",
```
Comment (cwjohnston) [inline content/sensu-go/6.2/api/events.md]: The change in https://github.com/sensu/sensu-go/pull/4111 is limited only to logging context. `id` key should stay the same in API responses. I've verified this in QA:

```
$ curl -s https://caviar.tf.sensu.io:8080/api/core/v2/namespaces/default/events -H "Authorization: Key $SENSU_API_KEY" | jq '.[0]|keys'   
[
  "check",
  "entity",
  "id",
  "metadata",
  "sequence",
  "timestamp"
]
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/6.2/api/events.md (hunk 2 above)
Comment (hillaryfraley) [reply content/sensu-go/6.2/api/events.md]: Got it, thank you

Comment (hillaryfraley) [inline content/sensu-go/6.2/api/events.md]:
```suggestion
    "id": "caaf2c38-2afb-4f96-89b3-8ca5c3e6f449",
```

File: content/sensu-go/6.2/api/events.md [hunk 3]
```diff
@@ -112,7 +113,8 @@ output         | {{< code shell >}}
...
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/api/events.md]:
```suggestion
    "id": "caaf2c38-2afb-4f96-89b3-8ca5c3e6f449",
```

File: content/sensu-go/6.2/api/events.md [hunk 4]
```diff
@@ -272,7 +274,8 @@ HTTP/1.1 200 OK
 [
   {
     "timestamp": 1543871497,
-    "id": "a68906e0-7c5c-49f0-8424-59a71d3ecfe2",
+    "event_id": "a68906e0-7c5c-49f0-8424-59a71d3ecfe2",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/api/events.md]:
```suggestion
    "id": "a68906e0-7c5c-49f0-8424-59a71d3ecfe2",
```

File: content/sensu-go/6.2/api/events.md [hunk 5]
```diff
@@ -323,7 +326,8 @@ HTTP/1.1 200 OK
   },
   {
     "timestamp": 1543871524,
-    "id": "095c37e8-1cb4-4d10-91e9-0bdd55a4f35b",
+    "event_id": "095c37e8-1cb4-4d10-91e9-0bdd55a4f35b",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/api/events.md]:
```suggestion
    "id": "095c37e8-1cb4-4d10-91e9-0bdd55a4f35b",
```

File: content/sensu-go/6.2/api/events.md [hunk 6]
```diff
@@ -387,7 +391,8 @@ output               | {{< code json >}}
 [
   {
     "timestamp": 1543871524,
-    "id": "095c37e8-1cb4-4d10-91e9-0bdd55a4f35b",
+    "event_id": "095c37e8-1cb4-4d10-91e9-0bdd55a4f35b",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/api/events.md]:
```suggestion
    "id": "095c37e8-1cb4-4d10-91e9-0bdd55a4f35b",
```

File: content/sensu-go/6.2/api/events.md [hunk 7]
```diff
@@ -455,7 +460,8 @@ HTTP/1.1 200 OK
 
 {
     "timestamp": 1577724113,
-    "id": "cf3c9fc0-023a-497a-aaf4-880dbd490332",
+    "event_id": "cf3c9fc0-023a-497a-aaf4-880dbd490332",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/api/events.md]:
```suggestion
    "id": "cf3c9fc0-023a-497a-aaf4-880dbd490332",
```

File: content/sensu-go/6.2/api/events.md [hunk 8]
```diff
@@ -545,7 +551,8 @@ response codes       | <ul><li>**Success**: 200 (OK)</li><li> **Missing**: 404 (
 output               | {{< code json >}}
 {
     "timestamp": 1577724113,
-    "id": "cf3c9fc0-023a-497a-aaf4-880dbd490332",
+    "event_id": "cf3c9fc0-023a-497a-aaf4-880dbd490332",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/api/events.md]:
```suggestion
    "id": "cf3c9fc0-023a-497a-aaf4-880dbd490332",
```

File: content/sensu-go/6.2/observability-pipeline/observe-events/events.md [hunk 9]
```diff
@@ -116,6 +116,8 @@ Sensu events contain:
   - Time that the event occurred in seconds since the Unix epoch
 - `event_id`
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/observability-pipeline/observe-events/events.md]:
```suggestion
- `id`
```

File: content/sensu-go/6.2/observability-pipeline/observe-events/events.md [hunk 10]
```diff
@@ -338,7 +340,8 @@ example      | {{< code shell >}}
     ]
   },
   "timestamp": 1552506033,
-  "event_id": "431a0085-96da-4521-863f-c38b480701e9"
+  "event_id": "431a0085-96da-4521-863f-c38b480701e9",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/observability-pipeline/observe-events/events.md]:
```suggestion
  "id": "431a0085-96da-4521-863f-c38b480701e9",
```

File: content/sensu-go/6.2/observability-pipeline/observe-events/events.md [hunk 11]
```diff
@@ -835,6 +847,7 @@ spec:
     user: agent
   timestamp: 1552594758
   event_id: 3a5948f3-6ffd-4ea2-a41e-334f4a72ca2f
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/observability-pipeline/observe-events/events.md]:
```suggestion
  id: 3a5948f3-6ffd-4ea2-a41e-334f4a72ca2f
```

File: content/sensu-go/6.2/observability-pipeline/observe-events/events.md [hunk 12]
```diff
@@ -954,7 +967,8 @@ spec:
       "user": "agent"
     },
     "timestamp": 1552594758,
-    "event_id": "3a5948f3-6ffd-4ea2-a41e-334f4a72ca2f"
+    "event_id": "3a5948f3-6ffd-4ea2-a41e-334f4a72ca2f",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/observability-pipeline/observe-events/events.md]:
```suggestion
    "id": "3a5948f3-6ffd-4ea2-a41e-334f4a72ca2f",
```

File: content/sensu-go/6.2/observability-pipeline/observe-events/events.md [hunk 13]
```diff
@@ -1199,7 +1214,8 @@ spec:
       ]
     },
     "timestamp": 1552506033,
-    "event_id": "431a0085-96da-4521-863f-c38b480701e9"
+    "event_id": "431a0085-96da-4521-863f-c38b480701e9",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/observability-pipeline/observe-events/events.md]:
```suggestion
    "id": "431a0085-96da-4521-863f-c38b480701e9",
```

File: content/sensu-go/6.2/observability-pipeline/observe-events/events.md [hunk 14]
```diff
@@ -1270,6 +1286,7 @@ spec:
       value: 0.004
   timestamp: 1552506033
   event_id: 47ea07cd-1e50-4897-9e6d-09cd39ec5180
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/observability-pipeline/observe-events/events.md]:
```suggestion
  id: 47ea07cd-1e50-4897-9e6d-09cd39ec5180
```

File: content/sensu-go/6.2/observability-pipeline/observe-events/events.md [hunk 15]
```diff
@@ -1353,7 +1370,8 @@ spec:
       ]
     },
     "timestamp": 1552506033,
-    "event_id": "47ea07cd-1e50-4897-9e6d-09cd39ec5180"
+    "event_id": "47ea07cd-1e50-4897-9e6d-09cd39ec5180",
```
Comment (hillaryfraley) [inline content/sensu-go/6.2/observability-pipeline/observe-events/events.md]:
```suggestion
    "id": "47ea07cd-1e50-4897-9e6d-09cd39ec5180",
```

File: content/sensu-go/6.2/api/events.md (hunk 2 above)
Comment (hillaryfraley) [reply content/sensu-go/6.2/api/events.md]: We are using event_id in the event references back to 5.20 -- I'm guessing these also need to be updated to id

**Review by echlebek — APPROVED** [review]
Looks good!

**Pull request merged**